experiments/keithley_sweep.py

```diff
@@ -47,8 +47,7 @@ def keithley_sweep(
     # initialize logging
     log = Log(
         log_file,
-        'NIai',
-        'V',
+        [dict(name='NIai', units='V')],
```

Comment:
I believe this should work but did we test it to make sure? Let's do this before merging.

Reply:
It works!